src/wooden-tree/stories/WoodenTree.stories.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { storiesOf } from '@storybook/react';
+import { store, WoodenTreeExample } from './WoodenTreeExample';
+
+storiesOf('WoodenTree', module)
+  .add('WoodenTreeExample', () => <Provider store={store}><WoodenTreeExample /></Provider>);
```

Comment:
I have a small nitpick here. Storybook should allow you to copy and paste the code to your project and give you a working component. Instead pf `<Provider store={store}><WoodenTreeExample /></Provider>` you should add whole markup inside `import { store, WoodenTreeExample } from './WoodenTreeExample';`

Reply:
I am sorry, but I don't get what you mean? What markup?

Reply:
The actual code from WoodenTreeExample. You should be able to just copy/paste the stuff from storybook and have working component.

Now you see in the storybook this:  `<Provider store={store}><WoodenTreeExample /></Provider>`

And you should be able to see basically all the code from WoodenTreeExample:
```JSX
// callback definitions etc. Even the data structure possibly
<ReduxTree
  hierarchicalCheck
  showCheckbox
  multiSelect={false}
  preventDeselect
  allowReselect
   checkboxFirst
   connectedNode={ConnectedNode}
   nodeIcon="fa fa-file-o"
   callbacks={{
      onDataChange: this.onDataChange,
       lazyLoad: this.lazyLoad,
      }}
      />
// whatever else you need.
```
`storiesOf(..).add(...)` must return component but whatever code you write inside of this will be visible on the storybook page. I would put at minimum the callback definitions so we know what arguments they take and what they should return. That is crucial information.